srcpkgs/ezstream/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'ezstream'
+pkgname=ezstream
+version=1.0.2
+revision=1
+build_style=gnu-configure
+hostmakedepends="gettext-devel automake pkg-config libtool"
+makedepends="check-devel libshout-devel libxml2-devel taglib-devel"
+depends="libshout libxml2 taglib"
+short_desc="Command line source client for media streams"
+maintainer="Alex Childs <misuchiru03+github@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://icecast.org/ezstream"
+distfiles="https://downloads.xiph.org/releases/ezstream/ezstream-${version}.tar.gz"
+checksum=11de897f455a95ba58546bdcd40a95d3bda69866ec5f7879a83b024126c54c2a
+
+#if [ "${XBPS_LIBC}" = musl ]; then
+#	hostmakedepends+=" musl-legacy-compat"
+#fi
+
+if [ "${XBPS_TARGET_LIBC}" = musl ]; then
+	makedepends+=" musl-legacy-compat"
+else
+	makedepends+=" glibc-devel"
```

Comment:
why is this needed?

Reply:
glibc-devel was there for `sys/queue.h`

Reply:
glibc-devel should be installed already for glibc-targeting builds

Reply:
Gotcha. Removed.